psydac/api/feec.py

```diff
@@ -4,25 +4,47 @@
 from psydac.feec.derivatives       import Derivative_1D, Gradient_2D, Gradient_3D
 from psydac.feec.derivatives       import ScalarCurl_2D, VectorCurl_2D, Curl_3D
 from psydac.feec.derivatives       import Divergence_2D, Divergence_3D
-from psydac.feec.global_projectors import Projector_H1, Projector_Hcurl
+from psydac.feec.global_projectors import Projector_H1, Projector_Hcurl, Projector_H1vec
 from psydac.feec.global_projectors import Projector_Hdiv, Projector_L2
 from psydac.feec.pull_push         import pull_1d_h1, pull_1d_l2
-from psydac.feec.pull_push         import pull_2d_h1, pull_2d_hcurl, pull_2d_hdiv, pull_2d_l2
-from psydac.feec.pull_push         import pull_3d_h1, pull_3d_hcurl, pull_3d_hdiv, pull_3d_l2
+from psydac.feec.pull_push         import pull_2d_h1, pull_2d_hcurl, pull_2d_hdiv, pull_2d_l2, pull_2d_vec
+from psydac.feec.pull_push         import pull_3d_h1, pull_3d_hcurl, pull_3d_hdiv, pull_3d_l2, pull_3d_vec
 
 __all__ = ('DiscreteDerham',)
 
 #==============================================================================
 class DiscreteDerham(BasicDiscrete):
     """ Represent the discrete De Rham sequence.
+    Should be initialized via discretize_derham function in api.discretization.py
+    
+    Parameters
+    ----------
+
+    mapping : Mapping
+        The mapping from the logical space to the physical space of the discrete De Rham.
+
+    get_vec : Bool
+        True to also get the "Hvec" space discretizing (H1)^n vector fields
+        
```

Comment:
I find it strange that this flag should be called `get_vec`, given that the H1-vector space is passed as an input to the constructor of this class.

Reply:
Checking the last space is an elegant solution. :+1: